app/src/main/res/layout/pre_match.xml

```diff
@@ -74,16 +74,28 @@
                 android:text="@string/pre_team_num"
                 tools:ignore="RtlSymmetry" />
 
-            <EditText
-                android:id="@+id/edit_TeamToScout"
-                android:layout_width="0dp"
-                android:layout_height="wrap_content"
+            <Spinner
+                android:id="@+id/spinner_TeamToScout"
+                android:layout_width="387dp"
+                android:layout_height="44dp"
                 android:layout_weight="1"
-                android:textSize="24sp"
+                android:background="@color/white"
                 android:gravity="center"
-                android:inputType="number"
+                android:paddingStart="10dp"
+                android:spinnerMode="dropdown"
                 android:textColor="@color/cpr_bkgnd"
-                android:background="@color/white" />
+                android:textSize="24sp"
+                tools:ignore="RtlSymmetry" />
+
+            <ImageView
+                android:layout_width="wrap_content"
+                android:layout_height="22dp"
+                android:layout_alignParentEnd="true"
+                android:layout_centerVertical="true"
+                android:layout_gravity="center"
+                android:paddingEnd="10dp"
+                android:src="@mipmap/drop_down_arrow"
+                tools:ignore="ContentDescription" />
```

Comment:
This is what you should use to make it look like the others:

            <RelativeLayout
                android:layout_width="100dp"
                android:layout_height="wrap_content"
                android:layout_gravity="center_vertical"
                android:orientation="horizontal">

                <Spinner
                    android:id="@+id/spinner_TeamToScout"
                    android:spinnerMode="dropdown"
                    android:layout_width="match_parent"
                    android:layout_height="35dp"
                    android:background="@color/white"
                    android:gravity="end|center"
                    android:paddingStart="10dp"
                    tools:ignore="RtlSymmetry" />

                <ImageView
                    android:layout_width="wrap_content"
                    android:layout_height="wrap_content"
                    android:layout_alignParentEnd="true"
                    android:paddingEnd="10dp"
                    android:paddingStart="10dp"
                    android:layout_centerVertical="true"
                    android:layout_gravity="center"
                    android:src="@drawable/cpr_drop_down_arrow"
                    tools:ignore="ContentDescription" />

            </RelativeLayout>


and change text_MatchPadding to 
android:layout_width="481dp"

Reply:
What padding?

I don't see a **text_MatchPadding**.